CONFIGURE.md

```diff
@@ -78,8 +132,8 @@ MiNiFi supports YAML and JSON configuration formats. The desired configuration f
 
 ### Scheduling strategies
 Currently Apache NiFi MiNiFi C++ supports TIMER_DRIVEN, EVENT_DRIVEN, and CRON_DRIVEN. TIMER_DRIVEN uses periods to execute your processor(s) at given intervals.
-The EVENT_DRIVEN strategy awaits for data be available or some other notification mechanism to trigger execution. CRON_DRIVEN executes at the desired intervals
-based on the CRON periods. Apache NiFi MiNiFi C++ supports standard CRON expressions without intervals ( */5 * * * * ).
+The EVENT_DRIVEN strategy awaits for data be available or some other notification mechanism to trigger execution. EVENT_DRIVEN is the recommended strategy for non-source processors.
+CRON_DRIVEN executes at the desired intervals based on the CRON periods. Apache NiFi MiNiFi C++ supports standard CRON expressions without intervals ( */5 * * * * ).
```

Comment:
What does "without intervals" refer to?  I would guess it means "no ranges", but [the Cron unit tests](https://github.com/apache/nifi-minifi-cpp/blob/main/libminifi/test/unit/CronTests.cpp) show support for `"0 0-5 14 * * ?"`, `"0 15 10 ? * MON-FRI"` and `"0 15 10 ? * 6L 2002-2005"`.  Maybe this limitation is out of date?

Reply:
I'm not sure what was this referring to, but as it is not clear I removed this limitation in 4e4434732dac8f91122fc910ea067c7c752283c6

Reply:
I think the rest of the sentence was useful; I would only remove the "without intervals" part.

Reply:
Readded in 8333c11812fda11ca06c6fe99e902b1fd58e50a7